subsys/net/openthread/Kconfig

```
@@ -52,6 +52,12 @@ config OPENTHREAD_RADIO_WORKQUEUE_STACK_SIZE
 	default 640 if NRF_802154_SER_HOST
 	default 512
 
+if CC310_BACKEND
+choice CC3XX_LOCK_VARIANT
+	default CC3XX_ATOMIC_LOCK
+endchoice
+endif
```

In this case, this looks nicer and makes it easier to see the dependencies in menuconfig.
```suggestion
choice CC3XX_LOCK_VARIANT
	default CC3XX_ATOMIC_LOCK if CC310_BACKEND
endchoice
```